frontend/src/app/pages/marker-page/marker-page.component.ts

```diff
@@ -178,6 +203,24 @@ export class MarkerPageComponent implements OnInit {
         this.marker.setZoomHandler(this.zoomHandler);
     }
 
+    private printCurrentLabellingTime() {
+        setInterval(() => {
+            if (!!this.taskStatus) {
+                const newTime = new Date(Date.now() - this.taskStatus.labellingTime);
+                const minutes = newTime.getMinutes();
+                const seconds = newTime.getSeconds();
+
+                this.currentTime = `${minutes < 10 ? '0' + minutes : minutes}:${seconds < 10 ? '0' + seconds : seconds}`;
```

Comment:
🖖

Reply:
Yep, fast way to get 01:01 format as well as 11:11.